jobs/io/ndb_io_test.py

```diff
@@ -33,13 +32,14 @@
 
 class NdbIoTests(job_test_utils.PipelinedTestBase):
 
-    def setUp(self):
-        super(NdbIoTests, self).setUp()
-        self.datastoreio_stub = stub_io.DatastoreioStub()
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.query_everything = (
```

Comment:
This sounds like a verb. Maybe self.all_items_query or something like that?

Reply:
Done.